include/translated-object.php

```diff
@@ -219,7 +219,7 @@ public function get_translation( $id, $lang ) {
 	 * @param int|string|object $lang language ( term_id or slug or object )
 	 * @return bool|int the translation post id  or term id if exists, otherwise the post id or term id, false if the post has no language
 	 */
-	public function get( $id, $lang ) {
+	public function get( int $id, $lang ) {
 		$obj_lang = $this->get_language( $id ); // FIXME is this necessary?
```

Comment:
This may break a lot as it will cause a fatal error, so I guess it should be `$id = (int) $id` instead.